### PR TITLE
[#121] Prepare releng for 3.26.1

### PR DESCRIPTION
**Pull request by ruspl-afed**
Bulk increment to 3.26.1

### CODE REVIEW

**Review by wimjongman — APPROVED** [review]
LGTM

File: mylyn.github/org.eclipse.mylyn.github.core/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -11,8 +11,8 @@ Export-Package: org.eclipse.mylyn.internal.github.core;version="6.2.0";x-friends
  org.eclipse.mylyn.internal.github.core.gist;version="6.2.0";x-friends:="org.eclipse.mylyn.github.ui",
  org.eclipse.mylyn.internal.github.core.issue;version="6.2.0";x-friends:="org.eclipse.mylyn.github.ui",
  org.eclipse.mylyn.internal.github.core.pr;version="6.2.0";x-friends:="org.eclipse.mylyn.github.ui"
-Require-Bundle: org.eclipse.mylyn.tasks.core;bundle-version="[3.26.0,4.0.0)",
- org.eclipse.mylyn.commons.net;bundle-version="[3.26.0,4.0.0)",
+Require-Bundle: org.eclipse.mylyn.tasks.core;bundle-version="[3.26.1,4.0.0)",
```
Comment (BeckerFrank) [inline mylyn.github/org.eclipse.mylyn.github.core/META-INF/MANIFEST.MF:14]: I think we need to change the Bundle-Version to 6.2.1.qualifier for all Mylyn GitHub Artifacts

**Review by BeckerFrank — CHANGES_REQUESTED** [review]
org.eclipse.mylyn.cdt-feature version should be changed to 5.22.1
and
org.eclipse.mylyn.github.* to 6.2.1

**Comment by ruspl-afed** [comment]
> org.eclipse.mylyn.cdt-feature version should be changed to 5.22.1 and org.eclipse.mylyn.github.* to 6.2.1

Great catch!

**Comment by ruspl-afed** [comment]
@BeckerFrank the PR was updated to address your comments

**Review by BeckerFrank — APPROVED** [review]
LGTM

**Comment by ruspl-afed** [comment]
Let's hold it a bit and try what Ed @merks  is [offering here](https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/123#issuecomment-1537101195)

**Comment by ruspl-afed** [comment]
> Let's hold it a bit and try what Ed @merks is [offering here](https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/123#issuecomment-1537101195)

Ed @merks are we ready to try version increment facilities? If yes, what do we need to do so?

**Comment by merks** [comment]
@ruspl-afed 

One thing I would ask and suggest is that we try to produce a milestone build from the main branch **before** you make the version increments in the main branch.

I.e., after this job finishes producing yet another nightly build, do a manual "build with parameters" and choose milestone in the drop-down:

https://ci.eclipse.org/mylyn/job/GitHub-mylyn-Orga/job/org.eclipse.mylyn/job/main/

This way we can test later that publishing another milestone with higher versioned IUs automatically deletes all previous milestone builds for the lower version IUs.  This is essential behavior for keeping the site of the sites limited (like you see how the nightly builds are limited to 5 right now)...

**Comment by ruspl-afed** [comment]
> One thing I would ask and suggest is that we try to produce a milestone build from the main branch before you make the version increments in the main branch.

Done, https://download.eclipse.org/mylyn/updates/milestone/S202305081049/index.html

**Comment by wimjongman** [comment]
Nice!!!

**Comment by merks** [comment]
FYI, now the branding icon is there too.  Note that each simple repository has a link to the commit state of that build:

image

**Comment by ruspl-afed** [comment]
Looks perfect, Ed! What is the next step here?

**Comment by merks** [comment]
I don't know if you folks want to promote the milestone to a release?  I see you already have a release here so maybe that would be confusing?

https://download.eclipse.org/justj/?file=mylyn/drops/3.26.0/v20230503-2157

Of course the content of the repository in this new structure is slightly different, given the name changes as well as the additional feature that was missing.   

One advantage of publishing a release is that then this repository will exist so that downstream consumers could start using that

https://download.eclipse.org/mylyn/updates/release/latest

Note that publishing a release (doing a release build), while it does do a full build, it **does not promote** the results of that build, but rather mirrors the milestone/latest repository as the release repository.

Personally I think this is probably a good idea, but it's your call and and doing this just a matter of doing a build and choosing "release" as the build type...

**Comment by ruspl-afed** [comment]
I would test the release promotion with a service release 3.26.1
As a very minimum for 3.26.1 we can complete this PR and improve our target regarding 3rd party libs,

**Comment by merks** [comment]
That’s okay too. You’ll have to increment again afterwards.

**Comment by merks** [comment]
FYI, you can already see the version updates in the nightly builds:

image

**Pull request merged**